library/src/main/java/com/bumptech/glide/util/Util.java

```diff
@@ -190,6 +192,16 @@ public static boolean bothNullOrEqual(Object a, Object b) {
     return a == null ? b == null : a.equals(b);
   }
 
+  public static boolean bothModelsNullOrEqual(Object a, Object b) {
```

Comment:
`bothNullOrEquivalent`

Reply:
+1

Reply:
Actually bothModelsNullOrEquivalent is probably even better if that's not with @TWiStErRob  meant.

Reply:
I did not change as it's also Equals if Equivalent not implemented and put the 3 things sounded odd :( Or I add a javadoc to the function ?

Reply:
bothNullEqualOrEquivalent?

Long name here is fine, it's kind of an unusual function. If you think it's best as is, I'm fine with that though, it's hard to explain this in a single method name.